src/WebUI/Pods/PodsTable.tsx

```diff
@@ -1,4 +1,4 @@
-import { BaseComponent, css } from "@uifabric/utilities";
+import { BaseComponent, css, format } from "@uifabric/utilities";
```

Comment:
use azure devops utilities format [if you do not have any string resources] or localeformat [if you have string resources]